ui-test/src/Configuration.ts

```diff
@@ -1,11 +1,14 @@
-require('dotenv').config({path: __dirname + '/.env'});
+require('dotenv').config({path: __dirname + '/../.env'});
```

Comment:
If we run the scripts defined in the package.json, the .env file gets copied over to the `out` folder and so it will be in current directory.  But this will work too.